src/13_file_io.py

```diff
@@ -9,11 +9,19 @@
 # Print all the contents of the file, then close the file
 # Note: pay close attention to your current directory when trying to open "foo.txt"
 
-# YOUR CODE HERE
+foo_file = open('foo.txt', 'r')
+print(foo_file.read())
+foo_file.close()
```

Comment:
Nice! This is good, try this with a for-loop when you have a chance. It may come up on the Sprint Challenge.